web/src/components/MapOverlays.tsx

```diff
@@ -12,7 +12,7 @@ export default function MapOverlays() {
   const surveyEnabled = useFeatureFlag('feedback-micro-survey') && !hasSeenSurveyCard;
 
   return (
-    <div className="fixed bottom-4 right-4 z-20 flex flex-col items-end space-y-3">
+    <div className="fixed bottom-4 right-4 z-20 hidden flex-col items-end space-y-3 sm:flex">
       <Suspense>
         <FeatureFlagsManager />
       </Suspense>
```

Comment:
I think we need to wrap each of these in their own div so they only take up the exact width and high each of the elements require.

Reply:
I'm not sure this will help 🤔

Reply:
Wouldn't wrapping each of them in their individual divs ensure they don't cover any more of the screen than they should? Right now it forms a big (transparent) box around the extreme points.

Reply:
I tried wrapping in divs when you suggested and it still grew to fill the larger container

Reply:
Ah, that's unfortunate. Let's go with the current approach then.